src/types/login_url.rs

```diff
@@ -1,10 +1,23 @@
-#[derive(Debug, Deserialize, Hash, PartialEq, Eq, Clone)]
+/// This object represents a parameter of the inline keyboard button used to
+/// automatically authorize a user. Serves as a great replacement for the
+/// [Telegram Login Widget] when the user is coming from Telegram. All the user
+/// needs to do is tap/click a button and confirm that they want to log in:
+///
+/// <div align="center">
+///     <img src="https://core.telegram.org/file/811140015/1734/8VZFkwWXalM.97872/6127fa62d8a0bf2b3c" width=300 />
+/// </div>
```

Comment:
I think you had better do it via markdown

Reply:
But markdown doesn't support `align="center"`.

Reply:
Are you sure we need `align="center"`?

Reply:
Looks much prettier. It is also centralized in the official docs.